dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -110,6 +115,9 @@ const SiteSelectionForm = ({ filepath, data, uiSchema }: SiteSelectionForm) => {
   const [activePage, setActivePage] = useState(0);
 
   useEffect(() => {
+    getFormData().then((data?: FormState) => {
```

Comment:
This should be in its own useEffect that is only ran when the component first loads. There could be an argument that we want to reload the saved form data when the schema changes but since that doesnt currently affect the loaded data is should be separate.

We also already have a prop called data (probably should be called schema) that clashes with this. It works because of scoping but its bad practice since its hides the parent one. We should rename one or both of these to be more specific.

Reply:
Didn't realise you could have more than one useEffect, the more you know! 

I tried to rename the `data` prop to `schema` too but it didn't work so only renamed the new one